canonical-web-and-design/maas-ui
# feat(ui): display action title in header if action selected

## Done

- Display action title in header if action selected, for machine list, machine details and KVM details headers

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list and select some machines
- Check that the take action menu labels are unchanged
- Choose an action and check that the action title shows in the header instead of "Machines"
- Go to a machine details page and check the same things
- Go to the KVM details page of a LXD KVM and click the "Refresh" icon
- Check that the action title is displayed in the header
- Select some VMs and select an action from the hamburger dropdown
- Check that the VM action title shows up in the header

## Fixes

Fixes canonical-web-and-design/app-squad#250
